singhjashanjot/LogPeers
# Implement "Coming Soon" Page

**Description:**

This PR introduces a "Coming Soon" page to the LogPeers application, providing users with a visually appealing and engaging placeholder while we finalize the development of upcoming features. The new page communicates that exciting updates are on the way, ensuring users remain informed and engaged.

**Changes Made:**

- Created a new `Learnings` component for the "Coming Soon" page.
- Implemented a minimalist design featuring:
  - A simple yet elegant layout that resembles a browser interface.
  - Clear and concise text indicating "Coming Soon" to inform users about upcoming features.
  - A user-friendly button that allows users to return to the home page, improving navigation.

**Preview:**

![image](https://github.com/user-attachments/assets/5efe8cce-cf2e-41ec-9aea-bdc3f01d6010) *(Screenshot of the working feature)*

**Testing Instructions:**

1. Pull this branch and run the application.
2. Navigate to the `/learnings` route to view the "Coming Soon" page.
3. Ensure the layout and text render correctly on both desktop and mobile devices.
4. Test the button to confirm it navigates back to the home page as expected.

**Additional Notes:**

- The "Coming Soon" page is temporary and will be replaced with actual content once the corresponding features are developed.
- I welcome any feedback or suggestions for further improvements!

### Checklist:

- [x] Changes are well-documented in the code
- [x] Review requested from at least one other contributor
